server/test/durablejobs/jobs/AddOperatorToLeafAddressServiceAreaJobTest.java

```diff
@@ -0,0 +1,1004 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.ebean.DB;
+import io.ebean.Database;
+import io.ebean.SqlRow;
+import io.ebean.Transaction;
+import io.ebean.annotation.TxIsolation;
+import java.time.Instant;
+import models.JobType;
+import models.PersistedDurableJobModel;
+import models.ProgramModel;
+import org.junit.Test;
+import repository.ResetPostgres;
+import services.program.ProgramBlockDefinitionNotFoundException;
+import services.program.predicate.Operator;
+
+public class AddOperatorToLeafAddressServiceAreaJobTest extends ResetPostgres {
+
+  private final Database database = DB.getDefault();
+
+  @Test
+  public void run_verifyEligibilityDefinitionAddsOperatorPropertyToSingleRootLevelAddressNode()
+      throws ProgramBlockDefinitionNotFoundException, JsonProcessingException {
+    String blockDefinitionsBeforeJob =
+        """
+        [
+          {
+            "id": 1,
+            "name": "Screen 1",
+            "repeaterId": null,
+            "description": "Screen 1 description",
+            "hidePredicate": null,
+            "localizedName": {
+              "isRequired": true,
+              "translations": {
+                "en_US": "Screen 1"
+              }
+            },
+            "optionalPredicate": null,
+            "questionDefinitions": [
+              {
+                "id": 2513,
+                "optional": false,
+                "addressCorrectionEnabled": true
+              }
+            ],
+            "localizedDescription": {
+              "isRequired": true,
+              "translations": {
+                "en_US": "Screen 1 description"
+              }
+            },
+            "eligibilityDefinition": {
+              "predicate": {
+                "action": "ELIGIBLE_BLOCK",
+                "rootNode": {
+                  "node": {
+                    "type": "leafAddressServiceArea",
+                    "questionId": 2513,
+                    "serviceAreaId": "Seattle"
+                  }
+                }
+              }
+            }
+          },
+          {
+            "id": 2,
+            "name": "Screen 2",
+            "repeaterId": null,
+            "description": "Screen 2 description",
+            "hidePredicate": null,
+            "localizedName": {
+              "isRequired": true,
+              "translations": {
+                "en_US": "Screen 2"
+              }
+            },
+            "optionalPredicate": null,
+            "questionDefinitions": [
+              {
+                "id": 2514,
+                "optional": false,
+                "addressCorrectionEnabled": false
+              }
+            ],
+            "localizedDescription": {
+              "isRequired": true,
+              "translations": {
+                "en_US": "Screen 2 description"
+              }
+            }
+          }
+        ]
+        """;
+
+    // The only difference in this string is the addition to the "operator" property on
```

Comment:
**suggestion (non-blocking):** Not sure this would work, but (here and below) you could try to create a variable and put %s in the line where the operator is and do something like: 
```
String blockDefinitionsBeforeJob = String.format(baseBlockDefinitionTemplate, ""); 
String expectedBlockDefinitionsAfterUpdate = String.format(baseBlockDefinitionTemplate, "\"operator\": \"IN_SERVICE_AREA\","); 
```